pulp_python/app/serializers.py

```diff
@@ -22,6 +23,50 @@ class Meta:
         fields = ('name',)
 
 
+class DistributionDigestSerializer(serializers.ModelSerializer):
+    """
+    A serializer for the Distribution Digest in a Project Specifier
+    """
+    type = serializers.CharField(
+        help_text=_("A type of digest: i.e. sha256, md5")
+    )
+
+    digest = serializers.CharField(
+        help_text=_("The digest of the distribution")
+    )
+
+    class Meta:
+        model = python_models.DistributionDigest
+        fields = ('type', 'digest')
+
+
+class ProjectSpecifierSerializer(serializers.ModelSerializer):
+    """
+    A serializer for Python project specifiers
+    """
+
+    name = serializers.CharField(
+        help_text=_("A python project name.")
+    )
+
+    version_specifier = serializers.CharField(
```

Comment:
It would be good to add validation to the version specifiers, although it may be non-trivial enough to make a future task.

http://www.django-rest-framework.org/api-guide/serializers/#field-level-validation